lib/beaker/host_prebuilt_steps.rb

```diff
@@ -505,19 +519,26 @@ def set_env host, opts
           host.exec(Command.new("startsrc -g ssh"))
         end
 
-        #ensure that ~/.ssh/environment exists
-        host.exec(Command.new("mkdir -p #{Pathname.new(host[:ssh_env_file]).dirname}"))
-        host.exec(Command.new("chmod 0600 #{Pathname.new(host[:ssh_env_file]).dirname}"))
-        host.exec(Command.new("touch #{host[:ssh_env_file]}"))
-
-        #add the constructed env vars to this host
-        host.add_env_var('RUBYLIB', '$RUBYLIB')
-        host.add_env_var('PATH', '$PATH')
+        if host['platform'] !~ /windows/ or (host['platform'] =~ /windows/ and host.is_cygwin?)
+          #ensure that ~/.ssh/environment exists
+          host.exec(Command.new("mkdir -p #{Pathname.new(host[:ssh_env_file]).dirname}"))
+          host.exec(Command.new("chmod 0600 #{Pathname.new(host[:ssh_env_file]).dirname}"))
+          host.exec(Command.new("touch #{host[:ssh_env_file]}"))
+          #add the constructed env vars to this host
+          host.add_env_var('RUBYLIB', '$RUBYLIB')
+          host.add_env_var('PATH', '$PATH')
+        end
+        #add the env var set to this test host
         env.each_pair do |var, value|
           host.add_env_var(var, value)
         end
 
-        host.exec(Command.new("cat #{host[:ssh_env_file]}"))
+        # print out the working env
+        if host.is_cygwin?
```

Comment:
Yeah, just to have the env printed out in the logs to make it clear what state beaker is running in.